distributed/diagnostics/tests/test_memray.py

```diff
@@ -4,6 +4,7 @@
 
 memray = pytest.importorskip("memray")
 
+pytestmark = pytest.mark.extra_packages
```

Comment:
I added this marker to have the job run really quickly. We could also mark with memray explicitly. No strong preference.